homeassistant/components/blink/__init__.py

```diff
@@ -27,6 +34,13 @@
     {vol.Required(CONF_NAME): cv.string, vol.Required(CONF_FILENAME): cv.string}
 )
 SERVICE_SEND_PIN_SCHEMA = vol.Schema({vol.Optional(CONF_PIN): cv.string})
+SERVICE_BULK_DOWNLOAD_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_PATH): cv.string,
+        vol.Required("since"): cv.string,
```

Comment:
`since` should be a datetime validator.
```suggestion
        vol.Required("since"): cv.datetime,
```